### PR TITLE
Build storage proto in Rust

### PR DESCRIPTION
**Pull request by daviddrysdale**
Still need to fill in the provenance details in the first commit.

### CODE REVIEW

File: rust/oak/build.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+extern crate protoc_rust;
+
+fn main() {
+    protoc_rust::run(protoc_rust::Args {
+        out_dir: "src/proto",
+        input: &["../../third_party/google/grpc/status.proto"],
```
Comment (tiziano88) [inline rust/oak/build.rs]: could these be combined as multiple inputs in the same protoc invocation?

Comment (daviddrysdale) [reply rust/oak/build.rs]: Yep, done.

File: oak/proto/storage.proto [hunk 2]
```diff
@@ -18,7 +18,7 @@ syntax = "proto3";
 
 package oak;
 
-import "google/rpc/status.proto";
+import "google/grpc/status.proto";
```
Comment (tiziano88) [inline oak/proto/storage.proto]: https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto suggests that `rpc` was the correct import path?

Comment (daviddrysdale) [reply oak/proto/storage.proto]: Ah, this was from https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto but the googleapis version seems to be upstream/canonical.  Fixed the first commit to reference googleapis and restored the path.

**Pull request merged**